liferay-echo/liferay-portal
# LPS-186629 Change the styles of the width and height inputs in simulation panel

Motivation
=======
We want to make the styles of the width and height inputs in the simulation panel more similar to the ones we hve for fragments in the page editor.
[Link to story or ticket](https://issues.liferay.com/browse/LPS-186629)

[Figma](https://www.figma.com/file/rMl4SoQeKBGejQLG9B6QOD/LPS-183847---Experiences-and-Variations-on-View-Mode?type=design&node-id=47-43616&t=yUiOFzBjhEeN4vdk-0)


Steps to Verify:
----------------
1. Set feature.flag.LPS-186155=true in portal-ext.properties
1. Open the simulation panel and click on screen size custom
1. Check that the simulation preview changes the width and height correctly when changing these inputs
1. Set the FF to false and check that everything keeps working as before
